Matlab_Scripts/5_Depth_Colourless/Pilot/markerinfo.txt

```diff
@@ -1,16 +1,23 @@
-Sent By Unity	Picked up by Vamp	 Event		Recoded as
-2	5	ON 1 			 Depth 1	101
-3	9	ON 2 			 Depth 2	102
-4	13	ON 3  			 Depth 3	103	
-5	65	ON 4 			 Depth 4	104
-6	69	ON 5 			 Depth 5	105
-7	73	RESP 1 			 Depth 1	106
-8	77	RESP 2  		 Depth 2	107
-9	129	RESP 3 	       		 Depth 3	108
-10	133	RESP 4 			 Depth 4	109
-11	137	RESP 5 			 Depth 5	110
-12	141	OFF 1 			 Depth 1	111
-13	193	OFF 2  			 Depth 2	112	
-14	197	OFF 3			 Depth 3	113
-15	201	OFF 4 			 Depth 4	114
-16	205	OFF 5 			 Depth 5	115
+Unity generates a list of triggers (coloumn 1) 
+Which are recorded by the ViewPix monitor/Vamp (coloumn 2)
+Events are either ON (onset) Resp (response) OFF (offset) at one of 5 depths
+Triggers are recoded for convience (Coloumn 4)
```

Comment:
when in the code are the "recoded" make it clear that in the raw data they are coded as something, and that some script "x.m" renames them

Reply:
indicate in a plain english sentence how to tell which of the depths is the target and which are standards for any given data file